test/language/arguments-object/mapped/mapped-arguments-nonwritable-nonenumerable-nonconfigurable-descriptors-4.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2017 Caio Lima. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: Mapped arguments property descriptor change to non-writable, non-enumerable and non-configurable
+description: >
+    Change the  descriptor using [[DefineOwnProperty]] to
+    {writable: false, enumerable: false}, change argument[0]
+    value to 2 using [[DefineOwnProperty]] and then
+    change property descriptor to {configurable: false}.
+    The descriptor's enumerable property continues with its configured value.
+flags: [noStrict]
+esid: sec-arguments-exotic-objects-defineownproperty-p-desc
+includes: [propertyHelper.js]
+---*/
+
+function argumentsNonWritableNonEnumerableThenNonConfigurableWithInterveningDefineOwnPropertyDescriptor(a) {
+  Object.defineProperty(arguments, "0", {writable: false, enumerable: false});
+  Object.defineProperty(arguments, "0", {value: 2});
+  Object.defineProperty(arguments, "0", {configurable: false});
```

Comment:
this test, as others, can be simplified to get to the point you really want to test here.

The goal here is apparently to assert the final descriptor, the same mapping parts is already asserted in other test cases.